openverse_catalog/dags/util/loader/test_sql.py

```diff
@@ -576,29 +577,61 @@ def test_upsert_records_replaces_data(postgres_with_load_and_image_table, tmpdir
     TITLE_B = "Bobs Great Pic"
     META_DATA_B = '{"description": "Bobs cool picture"}'
 
-    load_data_query_a = (
-        f"INSERT INTO {load_table} VALUES("
-        f"'{FID}','{LAND_URL_A}','{IMG_URL_A}','{THM_URL_A}',"
-        f"'{FILESIZE}','{LICENSE_A}','{VERSION_A}',"
-        f"'{CREATOR_A}','{CREATOR_URL_A}','{TITLE_A}','{META_DATA_A}',"
-        f"'{TAGS}','{WATERMARKED}','{PROVIDER}','{SOURCE}','{INGESTION}',"
-        f"'{WIDTH_A}','{HEIGHT_A}'"
-        f");"
+    query_values = create_query_values(
+        {
+            col.FOREIGN_ID: FID,
+            col.LANDING_URL: LAND_URL_A,
+            col.DIRECT_URL: IMG_URL_A,
+            col.THUMBNAIL: THM_URL_A,
+            col.FILESIZE: FILESIZE,
+            col.LICENSE: LICENSE_A,
+            col.LICENSE_VERSION: VERSION_A,
+            col.CREATOR: CREATOR_A,
+            col.CREATOR_URL: CREATOR_URL_A,
+            col.TITLE: TITLE_A,
+            col.META_DATA: META_DATA_A,
+            col.TAGS: TAGS,
+            col.WATERMARKED: WATERMARKED,
+            col.PROVIDER: PROVIDER,
+            col.SOURCE: SOURCE,
+            col.INGESTION_TYPE: INGESTION_TYPE,
+            col.WIDTH: WIDTH_A,
+            col.HEIGHT: HEIGHT_A,
+        }
     )
+    load_data_query_a = f"""INSERT INTO {load_table} VALUES(
+       {query_values}
+       );"""
     postgres_with_load_and_image_table.cursor.execute(load_data_query_a)
     postgres_with_load_and_image_table.connection.commit()
     sql.upsert_records_to_db_table(postgres_conn_id, identifier, db_table=image_table)
     postgres_with_load_and_image_table.connection.commit()
 
-    load_data_query_b = (
-        f"INSERT INTO {load_table} VALUES("
-        f"'{FID}','{LAND_URL_B}','{IMG_URL_B}','{THM_URL_B}',"
-        f"'{FILESIZE}','{LICENSE_B}','{VERSION_B}',"
-        f"'{CREATOR_B}','{CREATOR_URL_B}','{TITLE_B}','{META_DATA_B}',"
-        f"'{TAGS}','{WATERMARKED}','{PROVIDER}','{SOURCE}','{INGESTION}',"
-        f"'{WIDTH_B}','{HEIGHT_B}'"
-        f");"
+    query_values = create_query_values(
+        {
+            col.FOREIGN_ID: FID,
+            col.LANDING_URL: LAND_URL_B,
+            col.DIRECT_URL: IMG_URL_B,
+            col.THUMBNAIL: THM_URL_B,
+            col.FILESIZE: FILESIZE,
+            col.LICENSE: LICENSE_B,
+            col.LICENSE_VERSION: VERSION_B,
+            col.CREATOR: CREATOR_B,
+            col.CREATOR_URL: CREATOR_URL_B,
+            col.TITLE: TITLE_B,
+            col.META_DATA: META_DATA_B,
+            col.TAGS: TAGS,
+            col.WATERMARKED: WATERMARKED,
+            col.PROVIDER: PROVIDER,
+            col.SOURCE: SOURCE,
+            col.INGESTION_TYPE: INGESTION_TYPE,
+            col.WIDTH: WIDTH_B,
+            col.HEIGHT: HEIGHT_B,
+        }
     )
+    load_data_query_b = f"""INSERT INTO {load_table} VALUES(
+        {query_values}
+        );"""
```

Comment:
These changes are great! I'm seeing a lot of this kind of line, so we could potentially have another utility function that generates the `INSERT` SQL string for us.